di_website/common/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
```

Comment:
They are all helper classes and methods, i'd prefer to leave them there and just check first before re-writing any code that does the same thing, consider it a library with redundant helper methods

Reply:
Exactly, redundant ... we may or may not use them. I'd rather you check in GNR, find what you need then add it to the project... it's what's proper. Otherwise we've got all this code or libraries whose purpose in the project is unclear... that's pollution.

Reply:
Removed unused code from common